main.go

```diff
@@ -3,83 +3,94 @@ package main
 import (
 	"database/sql"
 	"fmt"
+	"log"
 	"os"
 	"os/signal"
 	"regexp"
 	"strconv"
 	"syscall"
 
-	config "./config"
+	"github.com/SteveHNH/repbot-go/config"
 	"github.com/bwmarrin/discordgo"
+	"github.com/jedib0t/go-pretty/table"
 	_ "github.com/mattn/go-sqlite3"
 )
 
 const setRep = `UPDATE reputation SET rep = rep + 1, user = ? WHERE username = ?`
 const getRep = `SELECT rep FROM reputation WHERE username = ?`
 const initRep = `INSERT OR IGNORE into reputation (username, user, rep) values (?, ?, ?)`
+const getRank = `SELECT username, rep from reputation ORDER BY rep DESC, username ASC`
```

Comment:
I think this one is going to give you the user ID (12345) rather than the name. 
I built the database weird to begin with so user = "stephen", username= "2309843" 

Long term I need to redo the columns so username is the actual text name, while user_id would be the numerical

Reply:
AH, cool.  I was curious about that.  I didn't have a copy of the DB to reference. If you can share the DB layout, I will adjust mine to match.

Reply:
SCHEMA:
CREATE TABLE reputation (username TEXT PRIMARY KEY, rep INTEGER DEFAULT 0, user VARCHAR);

example entry:
username | rep | user
121353108524761091 | 77| SteveHNH